apps/crisis-unleashed/styles/shadows/faction-shadows.ts

```
@@ -0,0 +1,65 @@
+    };
+  };
+  
+  const rgb = hexToRgb(factionColor);
+  const rgbaBase = `rgba(${rgb.r}, ${rgb.g}, ${rgb.b}`;
+  
+  switch (intensity) {
+    case 'low':
+      return `0 0 10px ${rgbaBase}, 0.3)`;
+    case 'high':
+      return `0 0 10px ${rgbaBase}, 0.7), 0 0 20px ${rgbaBase}, 0.5), 0 0 30px ${rgbaBase}, 0.3)`;
+    case 'medium':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)